### ISSUE TITLE
hugo serve does not serve font awesome anymore

### ISSUE THREAD

**Issue by roidelapluie** [google/docsy#1281]
related to #1165 

It does not work for me:

```
GET http://127.0.0.1:1313/webfonts/fa-solid-900.woff net::ERR_ABORTED 404 (Not Found)
main.css:1 
```

![off_000](https://user-images.githubusercontent.com/291750/195803423-87a8f3c6-69b9-4b9f-b0c0-9227de01d2bf.png)

```
./hugo mod graph
github.com/o11ydev/prometheus-workshop github.com/google/docsy@v0.5.0
github.com/o11ydev/prometheus-workshop github.com/google/docsy/dependencies@v0.5.0
github.com/google/docsy/dependencies@v0.5.0 github.com/twbs/bootstrap@v4.6.1+incompatible
github.com/google/docsy/dependencies@v0.5.0 github.com/FortAwesome/Font-Awesome@v0.0.0-20210804190922-7d3d774145ac

```

**Comment by deining** [comment]
@roidelapluie: I just submitted PR #1282 which addressed the issue raised by you. Can you test this fix and report back, whether it cures the problem? Your input is appreciated.

**Comment by deining** [comment]
Reopening since this issued is not fully addressed with #1282. Follow-up ticket #1284 needs to be merged, to. After that merge, we can close this ticket.
